src/screens/TopicMemberScreen/index.js

```diff
@@ -150,6 +168,7 @@ const TopicMemberScreen = () => {
 
   const handleScrollEvent = React.useCallback(
     (event) => {
+      Keyboard.dismiss();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleScrollEvent` function now calls `Keyboard.dismiss()`. This seems like a good addition, as it will hide the keyboard when the user starts scrolling, which can improve the user experience.
```diff
+ Keyboard.dismiss();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->